assets/_sass/components/_social-share-icons.scss

```diff
@@ -0,0 +1,69 @@
+/*==================================
+=       Social Share Icons         =
+==================================*/
+
+.share {
+    $s: &;
+    $i: icon-facebook, icon-twitter, icon-email, icon-print;
+    display: flex;
+    padding: .5rem;
+
+    &__text {
+        @include btn;
```

Comment:
`@include btn` doesn't exist meaning this isn't possible to test. I think you meant `@extend %btn`